todo/pages/index.tsx

```diff
@@ -1,72 +1,207 @@
+    const updateTodo = (id: number) => {
+        setTodoList((prevTodo) => prevTodo.map((todo) => todo.id === id ? { id: todo.id, title: editText, editMode: false, enabled: todo.enabled } : todo));
+        setFlashMessages({ todoTitle: editText, message: 'が更新されました！' });
+        setEditText('');
+    }
+
+    // 削除処理=======
+    const disableTodo = (id: number, title: string) => {
+        const todoTitle = title;
+        setTodoList(todoList.map((todo) => todo.id === id ? { id: todo.id, title: todo.title, editMode: todo.editMode, enabled: false } : todo));
+        setFlashMessages({ todoTitle: todoTitle, message: 'が削除されました！' });
+    };
+
+    return (
+        <div>
+            <Head>
+                <title>TODO</title>
+                <meta name="description" content="これはTODO作成アプリです。" />
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
+            <header>
+                <p className={styles.title}>TODOリスト</p>
+            </header>
+            <main>
+                {buttonState === 'plus' ?
+                    <div>
+                        <Button
+                            onClick={enableMinusButton}
+                            className={styles.show}>追加
+                        </Button>
+                    </div>
+                    :
+                    <div>
+                        <Button
+                            onClick={enablePlusButton}
+                            className={styles.hide}>
+                            取消
+                        </Button>
+                    </div>
+                }
+                {(flashMessages) && <p>{flashMessages.todoTitle}{flashMessages.message}</p>}
+                {(buttonState === 'minus') &&
+                    <div>
+                        <form>
+                            <Input
+                                type="text"
+                                value={inputText}
+                                onChange={handleChangeInput}
+                                className={styles.input}
+                                required />
+                            <Button
+                                disabled={inputText === ""}
+                                onClick={() => addTodo()}
+                                className={styles.primary}>
+                                登録
+                            </Button>
+                        </form>
+                    </div>
+                }
+                <div>
+                    <table>
+                        <thead>
+                            <tr>
+                                <th>タイトル</th>
+                                <th></th>
+                                <th></th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {todoList.map((todo) => todo.enabled &&
+                                <tr key={todo.id}>
+                                    {(todo.editMode) ?
```

Comment:
`editMode`にbooleanがセットされることが分からないかな笑
```suggestion
                                    {todo.isEditMode ?
```